apps/web/pages/settings/organizations/new/index.tsx

```diff
@@ -0,0 +1,32 @@
+import Head from "next/head";
+
+import { CreateANewOrganizationForm } from "@calcom/features/ee/organizations/components";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import { WizardLayout } from "@calcom/ui";
+
+import PageWrapper from "@components/PageWrapper";
+
+const CreateNewOrganizationPage = () => {
+  const { t } = useLocale();
+  return (
+    <>
+      <Head>
+        <title>{t("set_up_your_organization")}</title>
+        <meta name="description" content={t("organizations_description")} />
+      </Head>
```

Comment:
we have a `<Meta>` component

Reply:
Resolved in the [correct PR](https://github.com/calcom/cal.com/pull/9184) for these changes, thanks!